wagtail/admin/tests/pages/test_move_page.py

```diff
@@ -51,6 +56,28 @@ def setUpTestData(cls):
         cls.root_page.add_child(instance=cls.unpublished_page)
         cls.unpublished_page.add_child(instance=sub_page)
 
+        # Create test pages using SimpleParentPage and SimpleChildPage
+        cls.parent_page_a = SimpleParentPage(
+            title="Parent Page A", slug="parent-page-a"
+        )
+        cls.section_a.add_child(instance=cls.parent_page_a)
+
+        cls.parent_page_b = SimpleParentPage(
+            title="Parent Page B", slug="parent-page-b"
+        )
+        cls.section_a.add_child(instance=cls.parent_page_b)
+
+        cls.parent_page_c = SimpleParentPage(
+            title="Parent Page C", slug="parent-page-c"
+        )
+        cls.section_a.add_child(instance=cls.parent_page_c)
+
+        cls.child_page_1 = SimpleChildPage(title="Child Page 1", slug="child-page-1")
+        cls.parent_page_a.add_child(instance=cls.child_page_1)
+
+        cls.child_page_2 = SimpleChildPage(title="Child Page 2", slug="child-page-2")
+        cls.parent_page_b.add_child(instance=cls.child_page_2)
```

Comment:
Could you add some comments as to the setups for each? Or even better, name the parent accordingly (e.g. `parent_page_c` -> `parent_page_with_no_children`, `parent_page_a` -> `parent_page_with_max_children`?)